roles/confluent.ksql/defaults/main.yml

```diff
@@ -50,3 +50,5 @@ ksql:
     ksql.streams.producer.max.block.ms: 9223372036854775807
     producer.interceptor.classes: io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
     consumer.interceptor.classes: io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+    ksql.logging.processing.topic.auto.create: true
```

Comment:
do you want these properties if ksql_log_streaming_enabled == false?

Reply:
Good catch!